pom.xml

```diff
@@ -125,14 +130,29 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.7</version>
+            <version>${slf4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>log4j-over-slf4j</artifactId>
+            <version>${slf4j.version}</version>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>jul-to-slf4j</artifactId>
-            <version>1.7.7</version>
+            <version>${slf4j.version}</version>
         </dependency>
-        
+        <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Are we overriding a transitive dependency here? Looks like we don't need log4j and if that is the case, can we exclude log4j from the dependency bringing the log4j?

Reply:
yes, this would set the version for the transitive dependencies

Reply:
I have excluded the transitive dependency. Please take a look